packages/commercetools/theme/middleware/authenticated.js

```diff
@@ -0,0 +1,5 @@
+export default async ({ $vsf, redirect }) => {
+  if (await $vsf.$ct.api.isGuest()) {
```

Comment:
remember that isGuest is true when you have an item in the cart or wishlist but you still not logged in

Reply:
is there a bug report for this?

Reply:
fixed